packages/client-web/src/lib/utils.tsx

```diff
@@ -49,6 +49,16 @@ export function getFileParts(path: string) {
   const filename = path.split("/").pop() ?? "";
   const parts = filename.split(".");
 
+  // After obtaining the file name without its extension,
+  // if the obtained "stub" is empty, return the whole filename
+  // This fixes an issue where file names starting with a dot (e.g. ".DS_Store")
+  // would crash the application
+  if (parts.length > 0) {
```

Comment:
I think we need to still handle cases such as `.my_hidden_dot_file.txt` with the current implementation
```suggestion
  if (parts.length === 2) {
```

Reply:
That implementation would crash with filenames that have no dots (e.g. "README"), let me change it to handle those 2 cases separately